src/pudl/output/eia860.py

```diff
@@ -239,12 +241,22 @@ def generators_eia860(pudl_engine, start_date=None, end_date=None):
     Returns:
         pandas.DataFrame: A DataFrame containing all the fields of the EIA 860
         Generators table.
-
     """
+    # pudl_settings = pudl.workspace.setup.get_defaults()
+    # pudl_engine = sa.create_engine(pudl_settings["pudl_db"])
+
     pt = pudl.output.pudltabl.get_table_meta(pudl_engine)
     # Almost all the info we need will come from here.
     gens_eia860_tbl = pt['generators_eia860']
     gens_eia860_select = sa.sql.select([gens_eia860_tbl, ])
+    # To get plant age
+    generators_entity_eia_tbl = pt['generators_entity_eia']
+    generators_entity_eia_select = sa.sql.select([
+        generators_entity_eia_tbl.c.plant_id_eia,
+        generators_entity_eia_tbl.c.generator_id,
+        generators_entity_eia_tbl.c.operating_date,
+        # generators_entity_eia_tbl.c.report_date
+    ])
```

Comment:
I'm actually surprised that there aren't more fields being pulled in from the generators entity table already -- I think the output routines pre-date the the entity harvesting that @cmgosnell implemented, and so I think we might have lost a bunch of fields from the non-entity table output. I'll create an issue to flag this for follow up: #593